test-support/page-object/properties/is-hidden.js

```diff
@@ -22,9 +25,22 @@ export function isHidden(selector, options = {}) {
     isDescriptor: true,
 
     get() {
-      let element = findElement(this, selector, options);
+      let element = findElement(this, selector, options),
+        result;
+
+      if (element.length > 0) {
+        if (options.multiple) {
+          result = !(Ember.A(element).any(function(e) {
+            return $(e).is(':visible')
+          }));
+        } else {
+          result = element.is(':hidden')
+        }
```

Comment:
This can be written as `elements.get().reduce((result, elem) => result && $(elem).is(':hidden'));` or something like that without testing `options.multiple`. The `multiple` condition was already tested by `findElement`